lib/cc/presenters/github_pull_requests_presenter.rb

```diff
@@ -13,7 +13,7 @@ def initialize(payload, repo_config)
       end
 
       def success_message
-        if @repo_config.pr_status_quality_stats?
+        if issue_counts_in_payload?
```

Comment:
The way these issues are passed and particularly how they are set in this class means they could be nil. I figured it was worth it to keep this check so that if the counts aren't in the request, we can still report statuses on github PRs.